js/stores/windowStore.js

```diff
@@ -750,6 +752,22 @@ const doAction = (action) => {
         }
         break
       }
+    case windowConstants.WINDOW_ON_TOGGLE_BOOKMARK:
+      {
+        // show the AddEditBookmarkHanger control; saving/deleting takes place there
+        const activeFrame = frameStateUtil.getActiveFrame(windowState)
+        let siteDetail = siteUtil.getDetailFromFrame(activeFrame, siteTags.BOOKMARK)
+        const key = siteUtil.getSiteKey(siteDetail)
+
+        if (key !== null) {
```

Comment:
Not sure that we can to that, because now in your example you overrode sites from window state, which means that bookmark will not have favicon, tags, themeColor etc. It will only have `parentFolderId`, `title` and `customTitle`